tribuo-tutorials/client/src/main/kotlin/com/r3/conclave/samples/tribuo/client/EnclaveInstanceInfoChecker.kt

```diff
@@ -0,0 +1,26 @@
+package com.r3.conclave.samples.tribuo.client
+
+import com.r3.conclave.client.EnclaveConstraint
+import com.r3.conclave.common.EnclaveInstanceInfo
+
+/**
+ * This class is responsible for checking the [EnclaveInstanceInfo] for debug and simulation modes.
+ */
+open class EnclaveInstanceInfoChecker {
+    /**
+     * Check the [EnclaveInstanceInfo] against the [EnclaveConstraint].
+     * @param enclaveInstanceInfo [EnclaveInstanceInfo] to verify.
+     */
+    open fun check(enclaveInstanceInfo: EnclaveInstanceInfo) {
+        EnclaveConstraint.parse("S:4924CA3A9C8241A3C0AA1A24A407AA86401D2B79FA9FF84932DA798A942166D4 PROD:1 SEC:INSECURE").check(enclaveInstanceInfo)
```

Comment:
This should be passed into the client via the command line. This example constraint can be documented in the README